src/main/java/tasks/Event.java

```diff
@@ -0,0 +1,57 @@
+package tasks;
+
+import java.time.LocalDateTime;
+
+import common.DateTimeParser;
+
+public class Event extends Task {
+    private String from;
+    private String to;
+    private LocalDateTime fromDate;
+    private LocalDateTime toDate;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+        try {
+            this.fromDate = DateTimeParser.parse(from);
+        } catch (IllegalArgumentException e) {
+            //No correct format, deadline only recorded as a String
+            this.fromDate = null;
```

Comment:
Would it be a better idea to reprompt the user for input instead of setting the `fromDate`/`toDate` as `null`? This way working with DateTimes would be easier and you can allow for more functionality (like searching for tasks on a day).